gtsam/navigation/CombinedImuFactor.h

```diff
@@ -212,8 +212,6 @@ class GTSAM_EXPORT PreintegratedCombinedMeasurements : public PreintegrationType
    * sensor)
    * @param measuredOmega Measured angular velocity (as given by the sensor)
    * @param deltaT Time interval between two consecutive IMU measurements
```

Comment:
Please update the documentation for this parameter as well?

Reply:
done